syntax = "proto3";

package DTCC;

message Vector2D
{
  float x = 1;
  float y = 2;
}

message Polygon
{
  repeated Vector2D vertices = 1;
}

message Building
{
  string uuid = 1;
  Polygon footPrint = 2;
  double height = 3;
  double groundHeight = 4;
}

message CityModel
{
  repeated Building buildings = 4;
}
